docs/UserGuide.md

```diff
@@ -202,7 +203,7 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from the address book and schedule.
```

Comment:
should we clear tags with this

Reply:
Fixed in commit https://github.com/AY2122S2-CS2103-W17-1/tp/commit/89fc2cd4492d3dbe324cf58d65cb443a7b5f818b.